.bazelci/presubmit.yml

```diff
@@ -61,9 +81,54 @@ tasks:
     - "@bazel_workspace_b//:bin"
     - "@bazel_workspace_b//subdir:bin"
     build_targets:
-    - "//..."
+    - "//internal/..."
+    - "//packages/..."
     test_targets:
-    - "//..."
+    - "//internal/..."
+    - "//packages/..."
+  macos_e2e:
+    name: macos_e2e
+    platform: macos
+    # We need to reduce the memory & CPU usage of the top-level
+    # bazel process for bazel-in-bazel tests to not
+    # deplete the system memory completely.
+    # - startup JVM memory reduced
+    # - top-level bazel process should use as little memory as possible and only 1 core
+    # - nested bazel process should use a limited number of cores
+    shell_commands:
+    - echo 'startup --host_jvm_args=-Xms256m --host_jvm_args=-Xmx1536m' >> .bazelrc
+    build_targets:
+    - "//e2e/..."
+    test_flags:
+    - "--local_resources=792,1.0,1.0"
+    # test_args will be passed to the nested bazel process
+    - "--test_arg=--local_resources=8192,1.0,1.0"
+    test_targets:
+    - "//e2e/..."
+  macos_angular_bazel_example:
+    name: macos_angular_bazel_example
+    platform: macos
+    # We need to reduce the memory & CPU usage of the top-level
+    # bazel process for bazel-in-bazel tests to not deplete the
+    # system memory completely.
+    # - startup JVM memory reduced
+    # - top-level bazel process should use as little memory as possible and only 1 core
+    # - nested bazel process should use a limited number of cores
+    shell_commands:
+    - echo 'startup --host_jvm_args=-Xms256m --host_jvm_args=-Xmx1536m' >> .bazelrc
+    build_targets:
+    # Split up the build & test as test takes a long time and has very
+    # little output so running build first makes CI output friendlier.
+    # The build step can also use up more memory which may be required
+    # to build the release package with this configuration.
+    - "//e2e:e2e_angular_bazel_example"
+    test_flags:
+    - "--local_resources=792,1.0,1.0"
+    # test_args will be passed to the nested bazel process
```

Comment:
if you have anything in your head that would help us later when tuning the resources under CI, would be nice to have that in a README somewhere

Reply:
It's a long painful process tuning so that it passes consistently and still has enough cores to run quickly. There are differences on CircleCI and buildkite ubuntu, mac & Windows as well. I'll summarize what I found in a README.

@philwo Is there documentation somewhere on the memory & cores available on the different bazelci platforms? I didn't see it in the docs & did not see it anywhere obvious in the buildlkite UI.